GetService/handler.ts

```diff
@@ -88,21 +92,25 @@ export function GetServiceHandler(
   serviceModel: ServiceModel
 ): IGetServiceHandler {
   return async serviceId =>
-    (await serviceModel.findOneByServiceId(serviceId).run()).fold<
-      IGetServiceHandlerRet
-    >(
-      error => ResponseErrorQuery("Error while retrieving the service", error),
-      maybeService =>
-        maybeService.foldL<
-          IResponseErrorNotFound | IResponseSuccessJson<ServicePublic>
-        >(
-          () =>
-            ResponseErrorNotFound(
-              "Service not found",
-              "The service you requested was not found in the system."
-            ),
-          service => ResponseSuccessJson(retrievedServiceToPublic(service))
-        )
+    pipe(
+      await serviceModel.findOneByServiceId(serviceId)(),
+      E.foldW(
+        error =>
+          ResponseErrorQuery("Error while retrieving the service", error),
+        maybeService =>
+          pipe(
+            maybeService,
+            O.foldW(
+              () =>
+                ResponseErrorNotFound(
+                  "Service not found",
+                  "The service you requested was not found in the system."
+                ),
+              service => ResponseSuccessJson(retrievedServiceToPublic(service))
+            )
+          )
+      ),
+      x => x
```

Comment:
why this?

Reply:
Just for "type debugging" actually :D
I've removed it based on your and @AleDore's comments [here](https://github.com/pagopa/io-functions-app/pull/202/commits/c53ee9f5ec9a733ffe270aa44a05fa61ab8384fb)